lib/v2/udn/breaking-news.js

```diff
@@ -0,0 +1,90 @@
+const got = require('@/utils/got');
+const { parseDate } = require('@/utils/parse-date');
+const timezone = require('@/utils/timezone');
+const cheerio = require('cheerio');
+const { URL } = require('url');
```

Comment:
No need to import `URL`
```suggestion
const { art } = require('@/utils/render');
const path = require('path');
```